test/python/lattice_tight_binding.py

```diff
@@ -69,6 +74,35 @@ def test_TBLattice(self):
             tbl_read = arch['tbl']
 
         self.assertEqual(tbl, tbl_read)
+    
+    def test_TB_to_sympy_2D(self):
+        import sympy as sp
```

Comment:
We should maybe make this test optional to not depend on sympy? We could add something like we did in dft_tools:
```
        try:
            import sympy as sp
            from sympy import cos
        except ImportError:
            print('ImportError: Sympy needs to be installed to run this test)
            try:
                mpi.MPI.COMM_WORLD.Abort(1)
            except:
                sys.exit()
```

Reply:
Thank you for this recommendation. I appreciate it. It is fixed now.